### ISSUE TITLE
Emmet/Sublime tab key handler in .vue files when <style lang="*">

### ISSUE THREAD

**Issue by anyclub** [sergeche/emmet-sublime#657]
Emmet tab completions don't work with a language specified in the style tag. Works without a language specified. I'm specifically working with LESS styles but no other languages are working.

在VUE文件下，EMMET本可以正常工作
然后再style标签下设置lang=*后，TAB就没法工作了，但是ctrl+e却可以
欢迎中文回答~~
```
<style lang="less">
    body {
        m0  <-- press TAB here and nothing happens
    }
</style>
```

```
<style>
    body {
        m0  <-- press TAB here and works
    }
</style>
```

**Comment by artnikbrothers** [comment]
+

**Comment by reskwer** [comment]
Присоединяюсь, stylus совсем не работает, уже подумываю на чистом CSS писать :(

**Comment by sergeche** [comment]
Working on new Emmet version with better context detection

**Comment by artnikbrothers** [comment]
@reskwer you can use `cmd+e` instead of tab. I'm already used to it

**Comment by skyronic** [comment]
Hey @sergeche I'm a maintainer of the `vue-syntax-highlight` package. Here's some additional info which might help:

We recently switched to using `embed`/`extend` to include a different scope into a file. The default scope inside a `style` tag is:

```
<style lang="less">
<- text.html.vue source.css.less.embedded.html source.css.less
</style>
```

And without `lang`:

```
<style>
<- text.html.vue source.css.embedded.html source.css
</style>
```

Let me know if there's anything else I can do to help from my end.

**Comment by brownsugar** [comment]
I think this problem still exists, it's kind of inconvenient.

**Comment by BenHaldenby** [comment]
Yep, I'm also seeing this when tabbing in .vue files with `lang` on the style element: `<style lang="scss">`.
More complex abbreviations like `b0+p0` work (that gets expanded to:) ```border: 0;
padding: 0;``` . But simply `p0` with tab produces nothing. Any idea when a fix for this will land?

**Comment by linkhanfeng** [comment]
Emmet.subliem-settings
```
"syntaxProfiles": {
    "vue": "html"
  }
```
This is working for me!
